src/client/components/UserHeader.js

```diff
@@ -123,6 +124,7 @@ UserHeader.propTypes = {
   hasCover: PropTypes.bool,
   isFollowing: PropTypes.bool,
   onTransferClick: PropTypes.func,
+  isActive: PropTypes.bool.isRequired,
```

Comment:
We have two different styles of naming booleans in our codebase (mostly because of some legacy code and lack of consensus on that). Now it looks weird to have `active` and `isActive` represent the same thing. I think we should pick one version for booleans and use it from now on, and update old code as well. Not sure which one we should prefer.

@gregory-latinier @jm90m 

nit: usually we have required props first, then optional props.

Reply:
I'm used to write `isActive` and `setActive` from my past java coder life.

"Using the is prefix solves a common problem of choosing bad boolean names like status or flag. isStatus or isFlag simply doesn't fit,"

Reply:
Should be fine with if we include the prefix "is". isFlag could be isFlagged, isStatus does not make sense i think we would need to change naming, "status" does not feel like a boolean to me.

Reply:
it's the purpose of using `is` `flag` and `status` aren't good boolean name because `isStatus` doesn't make sense

Reply:
Let's try to use `isActive` or `hasChildren`, to avoid confusion with DOM attributes (which are single word). We would use this naming for all boolean variables (not only props) for consistency.